monitor.py

```diff
@@ -36,6 +36,18 @@ def fetch_release_date(release):
                     if "Created:" in p_ele:
                         start_date = p_ele.split(" ")[1]+" "+p_ele.split(" ")[2]
                         break
+                    else:
+                        prefix, number = release.rsplit(".", 1)
+                        prev_version = prefix + "." + str(int(number) - 1)
```

Comment:
Could you please check for all the case.
For eg: If my release is "4.20.0-ec.0". This will not be work as expected.